UPSTREAM_PROJECTS.yaml

```diff
@@ -133,6 +133,7 @@ projects:
           - tag: v1.24.1
           - tag: v1.25.0
           - tag: v1.26.0
+          - tag: v1.26.0
```

Comment:
Duplicated during generation bc we are temporarily using this tag for 1.26 and 1.27